api/base/utils.py

```diff
@@ -58,3 +59,9 @@ def waterbutler_url_for(request_type, provider, path, node_id, token, obj_args=N
 
     url.args.update(query)
     return url.url
+
+def node_token_creator(nodeid, userid):
```

Comment:
Parameters (and all variables) should use snake case. `nodeid` should be `node_id`.

ref: http://cosdev.readthedocs.org/en/latest/style_guides/python.html#naming